docs/add-ins/get-and-set-recurrence.md

```diff
@@ -0,0 +1,191 @@
+
+Office.context.mailbox.item.recurrence.setAsync(pattern, callback);
+
+function callback(asyncResult)
+{
+    console.log(JSON.stringify(asyncResult));
+}
+```
+
+## Get recurrence
+
+### Get recurrence as the organizer
+
+In the following example, in compose mode, the appointment organizer can get the recurrence object of an appointment series given the series or an instance of that series.
+
+```js
+Office.context.mailbox.item.recurrence.getAsync(callback);
+
+function callback(asyncResult){
+    var context = asyncResult.context;
+    var recurrence = asyncResult.value;
+
+    if (recurrence == null) {
+        console.log("Non-recurring meeting");
+        return;
+    }
+
+    console.log(JSON.stringify(recurrence));
+}
+```
+
+The following example shows the results of the `getAsync` call that retrieves the recurrence for a series.
+
+> [!NOTE]
+> In this example, `seriesTimeObject` is a placeholder for the JSON representing the `recurrence.seriesTime` property. You should use the [`SeriesTime`][SeriesTime link] methods to get the recurrence date and time properties.
+
+```json
+{
+    "recurrenceType": "weekly",
+    "recurrenceProperties": {
+        "interval": 1,
+        "days": ["tue","thu"],
+        "firstDayOfWeek": "sun"},
+    "seriesTime": {seriesTimeObject},
+    "recurrenceTimeZone": {
+        "name": "Pacific Standard Time",
+        "offset": -480}}
+```
+
+### Get recurrence as an attendee
+
+In the following example, an appointment attendee can get the recurrence object of an appointment series given the series, an instance of that series, or a meeting request.
+
+```js
+outputRecurrence(Office.context.mailbox.item);
+
+function outputRecurrence(item) {
+    var recurrence = item.recurrence;
+    var seriesId = item.seriesId;
+
+    if (recurrence == null) {
+        console.log("Non-recurring item");
+        return;
+    }
+
+    console.log(JSON.stringify(recurrence));
+}
+```
+
+The following example shows the value of the `item.recurrence` property for an appointment series.
+
+> [!NOTE]
+> In this example, `seriesTimeObject` is a placeholder for the JSON representing the `recurrence.seriesTime` property. You should use the [`SeriesTime`][SeriesTime link] methods to get the recurrence date and time properties.
+
+```json
+{
+    "recurrenceType": "weekly",
+    "recurrenceProperties": {
+        "interval": 1,
+        "days": ["tue","thu"],
+        "firstDayOfWeek": "sun"},
+    "seriesTime": {seriesTimeObject},
+    "recurrenceTimeZone": {
+        "name": "Pacific Standard Time",
+        "offset": -480}}
+```
+
+### Get the recurrence details
+
+After you've retrieved the recurrence object (either from the `getAsync` callback or from `item.recurrence`), you can get specific properties of the recurrence. For example, you can get the start and end dates and times of the series by using [methods][SeriesTime link] on the `recurrence.seriesTime` property.
```

Comment:
Is this correct? "[methods][SeriesTime link]" This looks like a placeholder. If not, it's missing the URL and the URL should be in (parens) not [square bracket].